tests/acceptance/github-compare-test.js

```diff
@@ -0,0 +1,41 @@
+import { run } from '@ember/runloop';
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+
+let container, store;
+
+moduleForAcceptance('Acceptance | github compare', {
+  beforeEach() {
+    container = this.application.__container__;
+    store = run(container, 'lookup', 'service:store');
+  }
+});
+
+test('finding a comparison without authorization', function (assert) {
```

Comment:
for consistency i included separate tests for authed/not authed but in general I'm not sure how useful constantly testing this difference.  It seems like we could test this once at the adapter level or something and get rid of all of this from the acceptance tests as they _aren't really_ testing that auth is working in any meaningful way unless I'm missing something

Reply:
Yea I don't have any strong preference on the auth/non-auth stuff.